front/src/assets/routes/index.ts

```diff
@@ -0,0 +1,19 @@
+import { MainPage, BasketPage } from "pages/index";
+
+type Route = {
+  path: string;
+  component: () => JSX.Element;
```

Comment:
zrób Route extends RouteProps, RouteProps zaimportowane z react-router-dom. Z reguly jak uzywasz jakiejs libki w standardowy sposob, typy do tego juz beda i musisz je znalezc w dokumentacji, albo hoverując nad propsami w tsx

Reply:
zrobiłem tak : 
interface Route extends RouteProps {
  path: string;
  component: () => JSX.Element;
  exact?: boolean;
}

export const mainRoutes: Route = ...

i "mainRoutes" podświetla się na czerwono i jest error : 
Type '({ path: string; component: () => Element; exact: boolean; } | { path: string; component: () => Element; })[]' is missing the following properties from type 'Route': path, componentts(2739)

Reply:
During